src/modules/m_timedbans.cpp

```diff
@@ -49,6 +49,7 @@ timedbans TimedBanList;
 class CommandTban : public Command
 {
 	ChanModeReference banmode;
+	bool send_notice;
```

Comment:
```suggestion
	bool sendnotice;
```

... and make this public.